src/editor/eventbuilder.cpp

```diff
@@ -2,85 +2,62 @@
 
 Event *EventBuilder::getEventNoModifier(QKeyEvent *e)
 {
-    if (e->key() == Qt::Key_BracketLeft)
+    switch(e->key())
     {
+    case Qt::Key_BracketLeft:
         return new EventBracketLeft();
-    }
-    if (e->key() == Qt::Key_Enter || e->key() == Qt::Key_Return)
-    {
+    case (Qt::Key_Enter | Qt::Key_Return):
         return new EventEnter();
-    }
-    if (e->key() == Qt::Key_Apostrophe)
-    {
+    case Qt::Key_Apostrophe:
         return new EventApostrophe();
-    }
-    if (e->key() == Qt::Key_Slash)
-    {
+    case Qt::Key_Slash:
         return new EventSlash();
+    default:
+        return new EventDefault();
     }
-    return new EventDefault();
 }
 
 Event *EventBuilder::getEventShiftModifier(QKeyEvent *e)
 {
-    if (e->key()==Qt::Key_BraceLeft)
+    switch(e->key())
     {
+    case Qt::Key_BraceLeft:
         return new EventBraceLeft();
-    }
-    if (e->key() == Qt::Key_ParenLeft)
-    {
+    case Qt::Key_ParenLeft:
         return new EventParenLeft();
-    }
-    if (e->key() == Qt::Key_QuoteDbl)
-    {
+    case Qt::Key_QuoteDbl:
         return new EventQouteDbl();
-    }
-    if (e->key() == Qt::Key_Enter || e->key() == Qt::Key_Return)
-    {
-        return new EventShiftEnter();
-    }
-    if (e->key() == Qt::Key_Asterisk)
-    {
+    case Qt::Key_Asterisk:
         return new EventAsterisk();
-    }
-    if (e->key() == Qt::Key_Space || e->key() == Qt::Key_Enter || e->key() == Qt::Key_Return)
-    {
+    case (Qt::Key_Enter | Qt::Key_Return):
```

Comment:
You better type here:
**case Qt::Key_Enter:
    return new EventShiftEnter();
case Qt::Key_Return:
    return newEventShiftEnter();**
And so on....